Code/Mantid/Framework/PythonInterface/mantid/api/src/Exports/IPeak.cpp

```diff
@@ -1,8 +1,8 @@
-#include "MantidAPI/IPeak.h"
+#include "MantidGeometry/Crystal/IPeak.h"
```

Comment:
This is now technically in the incorrect Python subpackage as it should be in `mantid.geometry`